boardgame-ui/src/components/timer/AddTimerForm.vue

```diff
@@ -0,0 +1,298 @@
+<template>
+  <div class="wrapper">
+    <b-loading :active="isLoading"/>
+    <form @submit.prevent="createTimer('form-timerCreation')" data-vv-scope="form-timerCreation" v-if="timer">
```

Comment:
```suggestion
    <form @submit.prevent="createTimer()" v-if="timer">
```
A scope is necessary for v-validate only if there are several forms in the page. If we leave the scope, the `errors.first(`user-${idx}`)` must be updated to correctly display the error messages.